### PR TITLE
[zh] Sync TOC election blog and ext-linked blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync TOC election blog and ext-linked blog into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/blog/2024/link-the-istio-service-mesh-for-people-who-have-stuff-to-do.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "外部文章：为繁忙的人们准备的 Istio 服务网格"
+description: 了解 Luca Cavallin 使用 Istio 的经验。
+publishdate: 2024-10-10
+attribution: Luca Cavallin - GitHub; Translated by Wilson Wu (DaoCloud)
+link: https://wilsonwu.me/blog/2024/the-istio-service-mesh-for-people-who-have-stuff-to-do/
```
Comment (windsonsea) [inline content/zh/blog/2024/link-the-istio-service-mesh-for-people-who-have-stuff-to-do.md:6]: `https://wilsonwu.me/` looks good, but is it suitable to link to a personal website?

File: content/zh/blog/2024/link-the-istio-service-mesh-for-people-who-have-stuff-to-do.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+{{< quote >}}
+我最近为开源服务网格项目 Istio 做出了一点贡献。我的贡献包括为 Istio CLI 命令之一添加一些测试。
+如果您想查看详细信息，可以在此处找到 Pull Request。
+这不是一个巨大的变化，但却是一次很棒的学习经历。
+在 Istio 上工作帮助我更深入地了解服务网格。我很高兴能做出更多贡献。
+在这篇文章中，我将解释 Istio 是什么、它为什么有用以及它是如何工作的。
+{{< /quote >}}
+
+[阅读英文版原文请访问 lucavall.in](https://www.lucavall.in/blog/the-istio-service-mesh-for-people-who-have-stuff-to-do)，
+[阅读中文版翻译文章请访问此处](https://wilsonwu.me/blog/2024/the-istio-service-mesh-for-people-who-have-stuff-to-do/)。
```
Comment (windsonsea) [inline content/zh/blog/2024/link-the-istio-service-mesh-for-people-who-have-stuff-to-do.md:18]: ditto

File: content/zh/blog/2024/toc-charter-elections/index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+* 为开发者和贡献者营造一个健康快乐的社区环境。
+
+虽然我们的供应商利益由指导委员会代表，但 TOC 成员资格与个人有关，
+无论其当前雇主是谁。成员以个人身份独立行事，
+必须优先考虑项目和社区的最佳利益。这始终是通过协商一致的方式实现的，
+因此我们的成员人数是相等的。TOC 传统上由 6 名成员组成，今后仍是如此。
+
+## 新章程有哪些变化？ {#what-changes-with-the-new-charter}
+
+The key changes in the [new charter](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md#charter), recently ratified by the Steering Committee, are:
```
Comment (windsonsea) [inline content/zh/blog/2024/toc-charter-elections/index.md:44]:
```suggestion
```

File: content/zh/blog/2024/link-the-istio-service-mesh-for-people-who-have-stuff-to-do.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/blog/2024/link-the-istio-service-mesh-for-people-who-have-stuff-to-do.md:6]: I think the ext-linked blog is as this way, #15770 also for the EN version is.

**Review by windsonsea — APPROVED** [review]
lgtm, cool!

**Pull request merged**